src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final int figures = 5;
+    private final int maxValue = 100;
```

Comment:
```suggestion
    private static final int FIGURES_COUNT = 5;
    private static final int MAX_VALUE = 100;
```

constant fields should look like this